lib/cipherstash/index/bloom_filter.rb

```diff
@@ -0,0 +1,28 @@
+require "openssl"
+
+module CipherStash
+  class Index
+    class BloomFilter
```

Comment:
Need to mark this class as `@private` (or make it a private constant) so as to avoid it turning up in the public API docs.